### PR TITLE
trans log2 log10 doc file

### PR DESCRIPTION
**Pull request by Joejiong**
trans log2.rst and log10.rst doc file to paddle/tensor/math/ folder

preview:
log10:
image
log2:
image

### CODE REVIEW

File: doc/paddle/api/alias_api_mapping [hunk 1]
```diff
@@ -346,8 +346,8 @@ paddle.tensor.math.maximum	paddle.maximum,paddle.tensor.maximum
 paddle.fluid.layers.add_position_encoding	paddle.nn.functional.add_position_encoding,paddle.nn.functional.extension.add_position_encoding
 paddle.nn.functional.common.cosine_similarity	paddle.nn.functional.cosine_similarity
 paddle.fluid.layers.floor	paddle.floor,paddle.tensor.floor,paddle.tensor.math.floor
-paddle.fluid.layers.log2    paddle.log2,paddle.tensor.log2,paddle.tensor.math.log2
-paddle.fluid.layers.log10    paddle.log10,paddle.tensor.log10,paddle.tensor.math.log10
+paddle.tensor.math.log2    paddle.log2,paddle.tensor.log2
```
Comment (TCChenlong) [inline doc/paddle/api/alias_api_mapping]: 这个地方的分隔符应该是 `\t` 所以目录展示错误了，应该展示在 paddle目录下

**Review by TCChenlong — APPROVED** [review]
LGTM
TODO：修改alias

File: doc/paddle/api/alias_api_mapping (hunk 1 above)
Comment (Joejiong) [reply doc/paddle/api/alias_api_mapping]: done

**Review by gfwm2013 — APPROVED** [review]
LGTM

**Pull request merged**